README.md

```diff
@@ -11,6 +11,7 @@ This is a basic application where you can write up your to do task and update it
 ## Getting Started - How to deploy on your machine
 
 - Clone the project
```

Comment:
- [OPTIONAL] Please kindly include the steps to clone the project and steps to run it locally.